thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/bao/AnomalyManagerImpl.java

```diff
@@ -340,6 +354,21 @@ public List<AnomalyDTO> decorate(final List<AnomalyDTO> l) {
     return outList;
   }
 
+  private List<AnomalyDTO> decorateWithFeedback(final List<AnomalyDTO> anomalies) {
+    final List<Long> feedbackIds = anomalies.stream()
+        .map(AnomalyDTO::getAnomalyFeedbackId)
+        .filter(Objects::nonNull)
+        .collect(Collectors.toList());
+
+    final List<AnomalyFeedbackDTO> feedbacks = genericPojoDao.get(feedbackIds, AnomalyFeedbackDTO.class);
+    final Map<Long, AnomalyFeedbackDTO> feedbackMap = new HashMap<>();
+    MapUtils.populateMap(feedbackMap, feedbacks, AnomalyFeedbackDTO::getId);
```

Comment:
Use java APIs here. I don't know whether this(import org.apache.commons.collections4.MapUtils) is at all required.